src/TLuaInterpreter.cpp

```diff
@@ -15449,6 +15450,71 @@ int TLuaInterpreter::getConnectionInfo(lua_State *L)
     return 2;
 }
 
+int TLuaInterpreter::unzip2(lua_State *L)
+{
+    QString zipLocation;
+    if (!lua_isstring(L, 1)) {
+        lua_pushfstring(L, "unzip2: bad argument #1 type (zip location as string expected, got %s!)", luaL_typename(L, 1));
+        return lua_error(L);
+    } else {
```

Comment:
:rotating_light: *return-after-else* anti-pattern detected here and for second argument... :wink:

Reply:
Fixed.